java/release_notes.txt

```diff
@@ -1,3 +1,14 @@
+May 03, 2016: libphonenumber-7.3.2
+Metadata changes:
+ - Updated phone metadata for region code(s):
+   AU, BR, ET, IN, KH, ML, NO, SB, TJ, US
+ - Updated short number metadata for region code(s): CA, ML, US
+ - New geocoding data for country calling code(s):
```

Comment:
@lararennie Can we change this back to what was automatically generated?

55_pt was actually new this time (it was removed in a recent release): https://github.com/googlei18n/libphonenumber/commits/master/java/geocoder/src/com/google/i18n/phonenumbers/geocoding/data/55_pt

And we've always included the entire 1xxx numbers according to the file names - see libphonenumber-7.1.1 release notes for example.